src/mesh/Mesh.hpp

```diff
@@ -49,6 +49,7 @@ class Mesh {
 
   /// A mapping from remote local ranks to the IDs that must be communicated
   using CommunicationMap = std::map<Rank, std::vector<VertexID>>;
+  using ConnectionMap    = CommunicationMap; // until we decide on a name
```

Comment:
because we currently use both? Should we try to resolve this here?

Reply:
I could, but this mostly unrelated to these changes.

Reply:
Then let's open an issue?

Reply:
https://github.com/precice/precice/issues/1539